llvm/test/DebugInfo/CAS/AArch64/test_relocation_zero.ll

```diff
@@ -1,35 +0,0 @@
-;RUN: llc -O0 --filetype=obj --cas-backend  --cas=%t/cas  --mccas-casid %s -o %t/test.id
```

Comment:
I don't think this test is needed because we store the DW_FORM_addr in the DistinctDataRef and no longer zero them out